### PR TITLE
Allow loose searches for scientific name

### PR DESCRIPTION
**Pull request by charvolant**
If requested and configured, check the scientific name to see if it is a vernacular name or taxon identifier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,6 +210,12 @@ Hints are used in two ways, if the server is configured to use them - see [below
   If hints are available, then the resulting match is checked against the list of hints and
   flagged with a `hintMismatch` issue if the match does not correspond to the hint.
 
+#### Loose matches
```
Comment (Rita-C) [inline README.md:213]: The word `loose` was not obvious to me as it's a loose search using the supplied scientific name, until I read the code:) 
Could be a bit unclear for other users as well?

File: server/src/main/java/au/org/ala/names/ws/resources/NameSearchResource.java [hunk 2]
```diff
@@ -439,6 +443,15 @@ private NameUsageMatch search(NameSearch search) throws Exception {
             result = metrics == null ? null : metrics.getResult();
         }
 
+        // See if the scientific name is actually a LSID
+        if (this.allowLoose && search.isLoose()) {
```
Comment (Rita-C) [inline server/src/main/java/au/org/ala/names/ws/resources/NameSearchResource.java:447]: I can see how`search.loose` is set up in the tests below, however, how/where `allowLoose` is configured?

Comment (charvolant) [reply server/src/main/java/au/org/ala/names/ws/resources/NameSearchResource.java:447]: See README.md line 243

File: README.md (hunk 1 above)
Comment (charvolant) [reply README.md:213]: Updated README.md

**Review by Rita-C — APPROVED** [review]

**Pull request merged**